src/DataStructures/DataBox/Tag.hpp

```diff
@@ -155,4 +155,34 @@ struct PrefixTag {};
  * \see DataBoxGroup SimpleTag
  */
 struct ComputeTag {};
+
+/*!
+ * \ingroup DataBoxGroup
+ * \brief Mark a struct as a reference tag by inheriting from this.
+ *
+ * \details
+ * A reference tag is used to identify an item in a DataBox that is a const
+ * reference to a subitem of a tagged container (such as a Variables or
+ * GlobalCache) in the DataBox
+ *
+ * \derivedrequires
+ * - type alias `base` that is the simple tag from which it is derived (and is
+ *   the tag used to fetch the subitem from the tagged container)
+ * - type alias `parent_tag` that is the tag for the tagged container
+ * - static function `get` that, given the item fetched by `parent_tag`, returns
+ *   a const reference to the subitem tagged by `base`
+ * - type alias `argument_tags` that is `tmpl::list<parent_tag>`
```

Comment:
It's arguably a little cumbersome to need to specify the parent class in two different ways -- is the use of `argument_tags` needed to fit into existing functionality built for compute tags?

Reply:
yes, it will make the DataBox logic simpler, and there will only be two Reference tags to start with...

Reply:
Sounds good :+1: -- I'm certainly in favor of simpler DataBox logic